styles/components/_lightsaber.scss

```diff
@@ -0,0 +1,36 @@
+.lightsaber {
+  align-items: center;
+  display: flex;
+  flex-basis: 100%;
+  order: -1;
+
+  &__handle {
+    height: 50px;
+    width: 300px;
+    background: url('../static/assets/ui/LightsaberHandle.png')
+      no-repeat center;
+    transform: translateX(7%);
```

Comment:
are you sure it's 7%?

Reply:
I find it unsafe, as it's parent doesn't have exact width set

Reply:
Yes, I have to think about changing this to something safer, but right now I want to implement this to next task. Can I fix it later?